app/models/recupload.rb

```diff
@@ -27,13 +28,19 @@ def acceptable_recletter
     return unless recletter.attached?
 
     unless recletter.blob.byte_size <= 20.megabyte
-      errors.add(:recletter, "is too big - file size cannot exceed 20Mbyte")
+      errors.add(:recletter, 'is too big - file size cannot exceed 20Mbyte')
     end
 
-    acceptable_types = ["image/png", "image/jpeg", "application/pdf"]
-    unless acceptable_types.include?(recletter.content_type)
-      errors.add(:recletter, "must be file type PDF, JPEG or PNG")
-    end
+    acceptable_types = ['image/png', 'image/jpeg', 'application/pdf']
+    return if acceptable_types.include?(recletter.content_type)
+
+    errors.add(:recletter, 'must be file type PDF, JPEG or PNG')
   end
 
+  def update_enrollment_status
+    enrollment = Recommendation.find(recommendation_id).enrollment
+    if !enrollment.application_fee_required || Payment.where(user_id: enrollment.user_id).current_camp_payments.exists?
```

Comment:
I think, status1_current_camp_payments scope should be used in line 42 instead of  current_camp_payments

Reply:
updated